app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,34 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = current_user.recipes.where(user_id: params[:food_id])
+  end
+
+  def new
+    @recipe = current_user.recipes.new(user_id: params[:food_id])
+  end
+
+  def create
+    @recipe = current_user.foods.new(recipe_params)
+
+    if @recipe.save
+      redirect_to user_food_recipes_url, notice: 'Recipe was successfully created.'
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @recipe = Recipe.find(params[:id])
+
+    if @Recipe.destroy
```

Comment:
- [ ] Kindly change the variable name from `@Recipe` to `@recipe` as it is case-sensitive.